lib/transform.js

```diff
@@ -48,6 +48,7 @@ function TreeTransformer(before, after) {
     this.before = before;
     this.after = after;
 }
+TreeTransformer.skip = MAP.skip;
 TreeTransformer.prototype = new TreeWalker;
```

Comment:
`MAP.skip` wasn't exposed in the exported API somewhere so it was added here.

Note that I originally implemented it as `TreeTransformer.prototype.skip = MAP.skip;` which made `--reduce-test` run twice as slow in Node 12. Apparently changing prototypes after creation causes V8 to disable some optimizations.

Reply:
No need for this hack - see https://github.com/mishoo/UglifyJS2/pull/3705/files#r375043705

Reply:
uglify-js API users ought to have access to this functionality. Right now there's no way to remove a node in a list.

Reply:
Also keep in mind that bin/uglifyjs needs this. It uses the public API.

Reply:
Please remove this line of change &minus; even if we decided to keep `TreeTransformer` exposed, it would make sense to expose `MAP` rather than this. And that should be done in a separate PR.

Reply:
I missed that one. Done.